### ISSUE TITLE
Suggestion: add qr as output 

### ISSUE THREAD

**Issue by StijnVA** [dan-da/hd-wallet-derive#51]
Adding a qr code as an output (ascii or image) would make the tool useful for an offline key store. For example storing the master key in an air gapped device and import only the derived keyes on mobile devices (using the camera) when needed.

**Comment by dan-da** [comment]
yeah good suggestion.   Especially since images can now be displayed in some terminals.

That said, this tool is mostly about displaying a list of addresses in table format, so it's unclear how one QR could be displayed per row.
